packages/runner/src/collect.ts

```diff
@@ -33,6 +33,7 @@ export async function collectTests(paths: string[], runner: VitestRunner): Promi
     try {
       const setupStart = now()
       await runSetupFiles(config, runner)
+      await loadDiffOptionFile(config, runner)
```

Comment:
Not sure if here is the right place to load diff config. Could we do this earlier? 🤔

Reply:
Is there a reason why you load it for each test file separately when there is only one config?

Reply:
Looks like I totally misunderstood "It should be loaded inside a test context" 😅

Reply:
Could you tell me where should I load this config file? Thanks!

Reply:
> inside a test context

It just means that we need to load it in the test runtime, not on the server side. I can recommend doing it in setup.common and then just pass this config down to the runner

Reply:
Now I got it, thanks for your patience and guidance!